src/constants/featureFlags.ts

```diff
@@ -1,3 +1,4 @@
 export const FEATURE_FLAGS = {
   GGS_WHITELISTED_REPOS: "ggs_whitelisted_repos",
+  IS_GGS_WHITELISTED: "is_ggs_whitelisted",
```

Comment:
just to be clear are we doing a whitelist or blacklist?

Reply:
changed naming to `IS_GGS_ENABLED` for clarity